### PR TITLE
feat(gateway): make forked pull-request builds optional

### PR DESCRIPTION
**Pull request by adamreese**
Makes building pull requests from forked repos optional.  Disabled by
default.

### CODE REVIEW

**Comment by adamreese** [comment]
Good point, let me look into that quick

**Comment by technosophos** [comment]
Re-LGTM with the env var change

**Comment by adamreese** [comment]
Merging this PR.  I will start a WIP for disabling at the project level

File: pkg/webhook/github.go [hunk 1]
```diff
@@ -23,21 +23,23 @@ const (
...
 
 // NewGithubHook creates a GitHub webhook handler.
-func NewGithubHook(s storage.Store) *githubHook {
+func NewGithubHook(s storage.Store, buildForkedPullRequests bool) *githubHook {
 	return &githubHook{
-		store:        s,
-		getFile:      getFileFromGithub,
-		createStatus: setRepoStatus,
+		store: s,
+		buildForkedPullRequests: buildForkedPullRequests,
```
Comment (cordoval) [inline pkg/webhook/github.go]: is not the second argument, is the fourth right?

Comment (bacongobbler) [reply pkg/webhook/github.go]: This is for initializing a struct, not a function. Fields are not positional.

**Pull request merged**